Probesys/bileto
# imp: Improve rendering of dates in the ticket timeline

## Related issue(s)



N/A

## Changes



- show the day in the timeline
- show the diff between two dates if diff >= 7 days
- show '(today)' next to the today date
- remove the date from the messages and events (only keep the hours:minutes)

## How to test manually



- open a ticket → check the date appears
- change the date of your computer
- add a comment at the new date → check the new date appears

## Checklist
